### PR TITLE
Repaths satchels and lets angels equip them

### PR DESCRIPTION
**Pull request by Shadowlight213**
Repaths satchels so all the different ones are children of the normal satchel.
Allows angels to equip satchels since the satchels are on the side of the body instead of the back. Should help miners who find the potion out a bit.
Merge token: #19225
:cl:
tweak: Angels can now equip satchels
/:cl:

### CODE REVIEW

**Comment by lzimann** [comment]
\>something that makes sense on ss13
REEEEEEEEE GET OUT
( :+1: )

**Comment by Shadowlight213** [comment]
Fucking maps. reee. Ok. I'll fix this once i finish doing achievments

**Comment by yackemflam** [comment]
>Keeps angels from using bags on their backs
>Lets them use bags of the same size anyhow

**Comment by MMMiracles** [comment]
@yackemflam satchels are the better looking backpack anyway, this only helps further empathize that point.

**Comment by ShadowDeath6** [comment]
backpack masterrace

**Comment by PKPenguin321** [comment]
Fuck satchels

**Comment by Cheridan** [comment]
time to nerf satchels tbh
they give you a smaller hitbox anyway

**Comment by optimumtact** [comment]
Please get this passing travis soon thanks

**Comment by Shadowlight213** [comment]
@optimumtact done

File: _maps/RandomZLevels/undergroundoutpost45.dmm [hunk 1]
```diff
@@ -4550,7 +4550,7 @@
 	initial_gas_mix = "co2=173.4;n2=135.1;plasma=229.8;TEMP=351.9";
 	heat_capacity = 1e+006;
 	name = "Cave Floor";
-	temperature = 363.9;
```
Comment (optimumtact) [inline _maps/RandomZLevels/undergroundoutpost45.dmm]: is removing all these colons significant?

Comment (Shadowlight213) [inline _maps/RandomZLevels/undergroundoutpost45.dmm]: I have no idea. the map merger did that by itself.

**Comment by optimumtact** [comment]
Lets hope the temps dont' explode

**Pull request merged**